src/views/datasets/DatasetDetailView.vue

```diff
@@ -72,4 +94,41 @@ onMounted(() => {
       :img="r.organization?.logo || r.owner.avatar"
     />
   </div>
+
+  <h2 class="fr-mt-4w">Discussions</h2>
+  <div v-if="!discussions.data?.length">Pas de discussion pour ce jeu de données.</div>
+
+  <DsfrAccordionsGroup>
```

Comment:
I'd find it easier to read if it was all expanded by default, with the possibility to collapse what we want. But that's more of a cosmetic change, not high priority.

Reply:
Unfortunately the accordion component does not support that: https://vue-dsfr.netlify.app/?path=/docs/composants-dsfraccordionsgroup--docs — only one item can be expanded at a time. We'll need to change the component used if we want a different look.